### PR TITLE
Feature/create initial feedback api

### PR DESCRIPTION
**Pull request by philratcliffe**
Added code for validating and processing POST request and writing it away to CosmodDB.

To test review the changes.

Review code changes
Run locally and ensure meets requirements
Run tests

### CODE REVIEW

File: PostFeedbackHttpTrigger/tests/fixtures/missing_is_useful.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "page": "course finder",
+    "questions": [{
+            "title": "postcodes are great",
+            "feedback": ""
+        },
+        {
+            "title": "you need a favicon",
+            "feedback": "see the title"
+
```
Comment (nshumoogum) [inline PostFeedbackHttpTrigger/tests/fixtures/missing_is_useful.json:10]: Unnecessary whitespace

File: PostFeedbackHttpTrigger/tests/fixtures/missing_is_useful.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+        }
+
```
Comment (nshumoogum) [inline PostFeedbackHttpTrigger/tests/fixtures/missing_is_useful.json:12]: And again

File: PostFeedbackHttpTrigger/tests/fixtures/valid_feedback.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    "page": "course finder",
+    "is_useful": true,
+    "questions": [{
+            "title": "postcodes are great",
+            "feedback": ""
+        },
+        {
+            "title": "you need a favicon",
+            "feedback": "see the title"
+
```
Comment (nshumoogum) [inline PostFeedbackHttpTrigger/tests/fixtures/valid_feedback.json]: I would tend to keep things a little tighter, no need for empty lines when the file is a json document

**Review by nshumoogum — APPROVED** [review]
Just a few comments

File: PostFeedbackHttpTrigger/tests/fixtures/missing_is_useful.json (hunk 1 above)
Comment (philratcliffe) [reply PostFeedbackHttpTrigger/tests/fixtures/missing_is_useful.json:10]: 👍

File: PostFeedbackHttpTrigger/tests/fixtures/missing_is_useful.json (hunk 2 above)
Comment (philratcliffe) [reply PostFeedbackHttpTrigger/tests/fixtures/missing_is_useful.json:12]: 👍

File: PostFeedbackHttpTrigger/tests/fixtures/valid_feedback.json (hunk 3 above)
Comment (philratcliffe) [reply PostFeedbackHttpTrigger/tests/fixtures/valid_feedback.json]: 👍
